filament/backend/src/vulkan/VulkanHandles.cpp

```diff
@@ -116,6 +116,9 @@ VulkanRenderTarget::VulkanRenderTarget(VulkanContext& context, uint32_t width, u
         if (color->target == SamplerType::SAMPLER_CUBEMAP) {
             viewInfo.viewType = VK_IMAGE_VIEW_TYPE_CUBE;
             viewInfo.subresourceRange.layerCount = 6;
+        } else if (color->target == SamplerType::SAMPLER_2D_ARRAY) {
+            viewInfo.viewType = VK_IMAGE_VIEW_TYPE_2D_ARRAY;
+            viewInfo.subresourceRange.layerCount = color->depth;
```

Comment:
I think we'll also need to set `baseArrayLayer` here, analogous to how we use `colorLevel` to set `baseMipLevel` above.

Reply:
Wow excellent catch!

Reply:
Fixed.